tests/unit/schema/test_schema.py

```diff
@@ -51,13 +52,35 @@ def test_select_by_tag():
 
     select_both = schema.select_by_tag("c")
     select_multi = schema.select_by_tag(["b", "c"])
-    select_neither = schema.select_by_tag("e")
+    select_neither = schema.select_by_tag("unknown")
 
     assert select_both == Schema([col1_schema, col2_schema])
     assert select_multi == Schema([col1_schema, col2_schema])
     assert select_neither == Schema([])
 
 
+def test_select_by_any_tag():
+    col1_schema = ColumnSchema("col1", tags=["a", "b", "c"])
+    col2_schema = ColumnSchema("col2", tags=["b", "c", "d"])
+    col3_schema = ColumnSchema("col3", tags=["b", "e", "f"])
+
+    schema = Schema([col1_schema, col2_schema, col3_schema])
+
+    col1_selection = schema.select_by_tag("a")
+    col2_selection = schema.select_by_tag("d")
```

Comment:
This should probably either be updated to `select_by_any_tag` or dropped (since it's repeating a test that already exists elsewhere.)